src/platforms/android/troubleshooting.mdx

```diff
@@ -73,6 +73,13 @@ For example, the following stack traces:
   at io.sentry.HubAdapter.captureEvent (HubAdapter.java:29)
 ```
 
+## Kotlin
+
+Stack trace elements referring to Kotlin inline functions from files different than the call site are incorrect, due to how they are compiled.
+A .class file with inline functions contains code from many source files. When the JVM generates a stacktrace, it will contain a different line number, because it doesn't substitute the class and line in which the error occurred with the original file and line number.
+As a result, if an exception is thrown inside an inline function, the line number of the stack trace of the Exception could be wrong.
+This is a missing feature in the JVM itself, so "Sentry can't fix this".
```

Comment:
```suggestion

Stack trace elements referring to Kotlin inline functions from files different than the call site may refer to the wrong line numbers.

This is due to a missing feature in the JVM. `.class` files with inline functions contain code from many source files, and when the JVM generates a stacktrace, it doesn't replace the class and line where the error occurred with the original file and line number. So, when an exception is thrown in an inline function, the line number for the stack trace of the Exception may be wrong.
```

Reply:
> Stack trace elements referring to Kotlin inline functions from files different than the call site may refer to the wrong line numbers.

Is this correct? I wanted the first sentence to tell users specifically what is wrong.

Reply:
yep, thanks!